src/main/java/org/folio/circulation/circulationrules/Match.java

```diff
@@ -1,4 +1,4 @@
-package org.folio.circulation.loanrules;
+package org.folio.circulation.circulationrules;
```

Comment:
What do you think to using `org.folio.circulation.rules` instead?

Reply:
I think this makes sense.

Reply:
Changed in https://github.com/folio-org/mod-circulation/commit/560bf2cf23e6da47be18ebf9a91df1228a2060a0